packages/caip/src/adapters/coingecko/utils.test.ts

```diff
@@ -34,6 +41,22 @@ const btc = {
   platforms: {}
 }
 
+const cosmos = {
+  id: 'cosmos',
+  symbol: 'atom',
+  name: 'Cosmos',
+  platforms: {
+    osmosis: 'IBC/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
+  }
```

Comment:
where did this come from? why is it called platforms?

Reply:
This is from https://api.coingecko.com/api/v3/coins/list?include_platform=true - I ditched the `binance-smart-chain` in those mocks since it's of no use to us currently and doesn't bring value to tests

Reply:
The need for `platforms` is coming from the implementation https://github.com/shapeshift/lib/commit/082a52a2fe46b1e4d16128800e75b6ed184aec1d#diff-94c505c2afc06d1d6c834988113d4e08aa259710cd7c69d06305d88227b40300R29 and allows for parsing of ERC20 in the context of Ethereum network.
When we will need support for CW20/CW721, we will be able to parse these similarly to what `parseEthData` currently does.

Reply:
ok. it makes more sense with the mock data named `makeCosmosMockCoingeckoResponse` and not just `btc`.